src/dlgTriggerEditor.cpp

```diff
@@ -27,6 +27,7 @@
 
 #include "dlgActionMainArea.h"
 #include "dlgAliasMainArea.h"
+#include "dlgButtonSelect.h"
```

Comment:
This is where you are going to have to start `#if`-ing things:

    #if QT_VERSION >=0x051000
        #include "dlgButtonSelect.h"
    #endif

I am not sure though whether it needs to be `0x050800`, `0x050900` or `0x051000` for inclusion only in 5.8 and later, 5.9 and later or 5.10 or later...